docs/sources/reference/components/prometheus.exporter.blackbox.md

```diff
@@ -40,6 +49,15 @@ The `config` argument must be a YAML document as string defining which blackbox_
 - `remote.http.LABEL.content`
 - `remote.s3.LABEL.content`
 
+The `targets` argument is an alternative to the [target][] block. This is useful when blackbox targets are supplied by another component.
```

Comment:
Is there some wording we can add here to make it clear that the one of either target or targets must be used (but not both). I'm not sure what are the best words though. @clayton-cornell

Reply:
sounds like both can be length zero below so this may not be true. I'll trust you all to think about/sort this question, but I think everything else is good to go now on the PR.

Reply:
You could be really straight forward and direct about it and add something like:

`You can't use both the `targets` argument and the `target` block in the same configuration file.`

Reply:
What about this:
```
You can't use both the `targets` argument and the `target` block in the same configuration file. The `targets` argument must be used when blackbox targets cannot be passed as a target block because another component supplies them.
```

Reply:
Yes... that makes it very clear when and where to use the argument.